### ISSUE TITLE
Rule `at-mixin-argumentless-call-parentheses` fails for mixin calls with content block arguments

### ISSUE THREAD

**Issue by jhae-de** [stylelint-scss/stylelint-scss#1184]
The `scss/at-mixin-argumentless-call-parentheses` rule does not correctly handle mixin calls with content block arguments and may fail report parentheses usage as expected. This leads to false negatives and inconsistent linting results.

**Stylelint configuration**
```javascript
{
  'scss/at-mixin-argumentless-call-parentheses': 'always',
}
```

**CSS code**
```css
@include my-mixin using($content-block-argument);
```

**Current behaviour**

The rule does not complain about the missing parentheses after the mixin name. Conversely, it also does not complain if the rule is configured with `never` and the mixin is called with parentheses.

**Expected behaviour**

The rule should report this as a problem:
```
Expected parentheses in mixin "my-mixin" call (scss/at-mixin-argumentless-call-parentheses)
```

The auto fix should result in:

```css
@include my-mixin() using($content-block-argument);
```

See [Sass documentation](https://sass-lang.com/documentation/at-rules/mixin/#passing-arguments-to-content-blocks).
